src/Sentry.Unity.Native/SentryNative.cs

```diff
@@ -17,59 +17,58 @@ public static class SentryNative
         /// Configures the native SDK.
         /// </summary>
         /// <param name="options">The Sentry Unity options to use.</param>
-        public static void Configure(SentryUnityOptions options)
+        /// <param name="sentryUnityInfo">Infos about the current Unity environment</param>
+        public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentryUnityInfo)
         {
-            if (ApplicationAdapter.Instance.Platform switch
+            if (!sentryUnityInfo.IsNativeSupportEnabled(options, ApplicationAdapter.Instance.Platform))
```

Comment:
the original check only worked on Windows and Linux, i.e. platforms where we bundle sentry-native. Now, `IsNativeSupportEnabled` returns true for a lot more platforms. Is that desirable? I guess it's fine because we don't trigger this code unless it's one of the supported platforms?

Reply:
The `IsNativeSupportEnabled` still does the check of the current runtime vs the flag on the options.
But yeah, we only get there if we support the platform in the first place.